templates/search/index.html.twig

```diff
@@ -0,0 +1,74 @@
+                        {{ knp_pagination_sortable(pagination, 'Complainant'|trans({}, 'case'), 'c.sortingComplainant') }}
+                    </th>
+                    <th{% if pagination.isSorted('c.SortingCounterparty') %} class="sorted"{% endif %}>
+                        {{ knp_pagination_sortable(pagination, 'Counterparty'|trans({}, 'case'), 'c.sortingCounterparty') }}
+                    </th>
+                    <th scope="col" data-orderable="false">{{ 'Function'|trans({}, 'case') }}</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for case in pagination %}
+                    <tr>
+                        <td><a href="{{ url('case_summary', {id: case.id}) }}" data-toggle="tooltip" data-placement="auto" title="{{ 'Show case'|trans({}, 'case') }} {{ case.casenumber|e }}">{{ case.casenumber|e }}</a></td>
+                        <td>{{ case.complaintCategory.name|e }}</td>
+                        <td>{{ case.currentPlace|e }}</td>
+                        <td>{{ case.sortingAddress|e }}</td>
+                        <td>{{ case.sortingComplainant|e }}</td>
+                        <td>{{ case.sortingCounterparty|e }}</td>
+                        <td>
+                            <div class="btn-group" role="group" aria-label="{{ 'List functions'|trans({}, 'case') }}">
+                                <a href="" role="button" class="btn btn-success" data-toggle="modal" data-target="#assign_case{{ case.id }}" data-placement="auto" title="{{ 'Assign caseworker'|trans({}, 'case') }}"><i class="fas fa-user-plus"></i><span class="sr-only">{{ 'Assign caseworker'|trans({}, 'case') }}</span></a>
+                                <a href="" role="button" class="btn btn-warning" data-toggle="modal" data-target="#reminder_new{{ case.id }}" data-placement="auto" title="{{ 'Add reminder'|trans({}, 'case') }}"><i class="fas fa-clock"></i><span class="sr-only">{{ 'Add reminder'|trans({}, 'case') }}</span></a>
+                                <a href="{{ url('case_summary', {id: case.id}) }}" role="button" class="btn btn-primary" data-toggle="tooltip" data-placement="auto" title="{{ 'Show case'|trans({}, 'case') }}"><i class="fas fa-eye"></i><span class="sr-only">{{ 'Show case'|trans({}, 'case') }}</span></a>
+                            </div>
+                        </td>
+                    </tr>
+                    {{ render(controller('App\\Controller\\CaseReminderController:new', {id: case.id})) }}
+                    {{ render(controller('App\\Controller\\CaseController:assignCaseworker', {id: case.id})) }}
+                {% endfor %}
+                </tbody>
+            </table>
+            <div class="navigation">
+                {{ knp_pagination_render(pagination) }}
+            </div>
+        </div>
+    {% else %}
+            {{ 'search_no_results'|trans({'%searchTerm%': search}, 'search') }}
```

Comment:
We usually use real texts for translations and not keys, i.e. “No results for %search%“ rather than ”search_no_results“.

Reply:
Fixed in 79cc63b0254e7d4d3b73c22d6e2fa743eae6e690.